instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/HttpServerTracer.java

```diff
@@ -49,11 +50,11 @@
 
   public static final String CONTEXT_ATTRIBUTE = "io.opentelemetry.instrumentation.context";
 
-  protected static final String USER_AGENT = "User-Agent";
-
   private static final boolean FAIL_ON_CONTEXT_LEAK =
       Boolean.getBoolean("otel.internal.failOnContextLeak");
 
+  protected static final String USER_AGENT = "User-Agent";
+
```

Comment:
may as well keep prior order here
```suggestion
  protected static final String USER_AGENT = "User-Agent";

  private static final boolean FAIL_ON_CONTEXT_LEAK =
      Boolean.getBoolean("otel.internal.failOnContextLeak");

```

Reply:
what is the benefit does it gain? I usually put protected behind private.